src/simplewallet/simplewallet.cpp

```diff
@@ -3956,6 +3956,19 @@ void simple_wallet::wallet_idle_thread()
   }
 }
 //----------------------------------------------------------------------------------------------------
+std::string simple_wallet::get_prompt() const
```

Comment:
Does this need to be `const`? This is being set in the `simple_wallet::run` method which is not `const`. And it is modifying several variables without a lock.